tarsier-service/service/src/llm.py

```diff
@@ -0,0 +1,31 @@
+from openai import OpenAI
+from dotenv import load_dotenv
+
+
+load_dotenv()
+client = OpenAI()
```

Comment:
not sure if this is still a problem but making multiple requests (especially concurrent requests) on a single open ai client used to be problematic. Should instantiate in the function it's being used in